flink-kubernetes/src/main/java/org/apache/flink/kubernetes/KubernetesResourceManagerDriver.java

```diff
@@ -360,7 +364,12 @@ public void onError(List<KubernetesPod> pods) {
 
         @Override
         public void handleFatalError(Throwable throwable) {
```

Comment:
maybe we should rename this method into `handleError` because it now also handles non fatal errors such as the `KubernetesTooOldResourceVersionException`.